src/main/java/zeus/main/TaskList.java

```diff
@@ -0,0 +1,103 @@
+package zeus.main;
+
+import java.util.ArrayList;
+
+import zeus.task.Task;
+
+
+/**
+ * Class that represents a list of tasks.
+ */
+public class TaskList {
+
+    private ArrayList<Task> taskList;
```

Comment:
```suggestion
    private ArrayList<Task> tasks;
```
Perhaps this could be in plural form since you are using a collection of objects.